tests/unit/lms/services/digest/_digest_assistant_test.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime, timedelta
+
+import pytest
+from h_matchers import Any
+
+from lms.models.lti_role import RoleScope, RoleType
+from lms.services.digest._digest_assistant import DigestAssistant
+from lms.services.digest._models import HCourse, HUser
+from tests import factories
+
+
+class TestDigestAssistant:
+    def test_get_h_users(self, assistant):
+        for days, (userid, email, display_name) in enumerate(
+            (
+                ("h_userid_1", None, None),
+                ("h_userid_1", "WRONG", "WRONG"),
+                ("h_userid_1", "email_1", "display_1"),
+                ("h_userid_1", None, None),
+                # Check we can get more than one at a time
+                ("h_userid_2", "email_2", "display_2"),
+            )
+        ):
+            factories.User(
+                h_userid=userid,
+                email=email,
+                display_name=display_name,
+                # Ensure the insert order is the updated order too
+                updated=datetime.now() + timedelta(days=days),
```

Comment:
I don't think this is required, as the insert order happens to do this for us, but this seems belt and braces? 

I can remove it if you don't think it's required.